### PR TITLE
Feat/horizontal nav

### PR DESCRIPTION
**Pull request by Alexandre-Herve**
Adding two components for a horizontal nav:
- menu (a astyled `ul`)
- link (to be used in side a `<Link>` tag)

Example of use:
snapshot600

### CODE REVIEW

File: src/components/horizontal-nav/horizontal-nav-menu/HorizontalNavMenu.module.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.menu {
+  align-items: center;
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  margin: 0;
+  padding: 0 0 8px 0;
+
+  a {
+    text-decoration: none;
```
Comment (oclero) [inline src/components/horizontal-nav/horizontal-nav-menu/HorizontalNavMenu.module.scss:10]: Do we keep the outline?

File: src/components/horizontal-nav/horizontal-nav-link/HorizontalNavLink.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import 'jest-styled-components';
```
Comment (lhoursquentin) [inline src/components/horizontal-nav/horizontal-nav-link/HorizontalNavLink.spec.tsx:3]: unused?

File: src/components/horizontal-nav/horizontal-nav-menu/HorizontalNavMenu.module.scss (hunk 1 above)
Comment (Alexandre-Herve) [reply src/components/horizontal-nav/horizontal-nav-menu/HorizontalNavMenu.module.scss:10]: I don't see any outline :shrug:

File: src/components/horizontal-nav/horizontal-nav-link/HorizontalNavLink.spec.tsx (hunk 2 above)
Comment (Alexandre-Herve) [reply src/components/horizontal-nav/horizontal-nav-link/HorizontalNavLink.spec.tsx:3]: A damn you're right, forgot to remove them :+1:

**Pull request merged**